packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 52.9.1 - August 12, 2021
```

Comment:
@sbudhirajadoc Are you available to review these changes?

Reply:
@rcoringrato-sfdc let's update later if needed when @sbudhirajadoc get to review it